README.md

```diff
@@ -305,6 +309,61 @@ For local development (or whenever you have a `*-dev` docker image deployed), th
 ./manage.py shell_plus
 ```
 
+### vs code Integration
+
+There are some possibilities to debug this codebase from within visual studio code.
+
+#### Debug from vs code
+
+In order to debug the service from within vs code, you need to create a launch-configuration. Create
+a folder `.vscode` in the root folder if it doesn't exist and put a file `launch.json` with this content
+in it:
+
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Python Debugger: Attach",
+      "type": "debugpy",
+      "request": "attach",
+      "justMyCode": false,
+      "connect": {
+        "host": "localhost",
+        "port": 5678
+      }
+    }
+  ]
+}
+
+```
```

Comment:
Could we add that to our ansible setup?

Reply:
It needs to be within the context of the repository. Do we have something there that I don't know yet?